package.json

```diff
@@ -17,17 +18,22 @@
     "@types/react-test-renderer": "^18.3.0",
     "expo": "^51.0.0",
     "expo-image": "~1.12.8",
+    "expo-linking": "~6.3.1",
```

Comment:
This isn't a change from this PR, but this is looking a little sus. Can you remove the package listed here and see if everything still works?
`"-": "^0.0.1",`

Reply:
removed `L14` `"-": "^0.0.1",`
I don't know how that got there and it is obviously unnecessary, as app works without it in `package.json` and after having trashed and reinstalled `node_modules`.

Reply:
> removed `L14` `"-": "^0.0.1",` I don't know how that got there and it is obviously unnecessary, as app works without it in `package.json` and after having trashed and reinstalled `node_modules`.

this came in during the typescript migration package

Reply:
I'll remove `"save-dev": "^0.0.1-security"` too.

Here's the link to the article that I followed re: TypeScript migration, as requested:
[https://medium.com/@sayalia72/seamless-migration-transforming-your-react-native-project-from-javascript-to-typescript-5be0f6ef1e08](url)